mcap_vendor/CMakeLists.txt

```diff
@@ -21,7 +21,8 @@ macro(build_mcap_vendor)
   include(FetchContent)
   fetchcontent_declare(mcap
     GIT_REPOSITORY https://github.com/foxglove/mcap.git
-    GIT_TAG 066f03b9b8444ffc1134e852e3419e4a4c0bef85 # releases/cpp/v0.6.0
+    # GIT_TAG 066f03b9b8444ffc1134e852e3419e4a4c0bef85 # releases/cpp/v0.6.0
```

Comment:
TODO: Target updated mcap release hash once the Windows-export change is merged and released

Reply:
https://github.com/foxglove/mcap/pull/711 has merged, did the release happen?

Reply:
Was waiting on followup foxglove/mcap#713. Release happened just now, and this is updated